karate-grpc-demo/src/test/java/demo/TestBase.java

```diff
@@ -1,21 +1,30 @@
 package demo;
 
-import org.junit.AfterClass;
+import com.intuit.karate.Results;
+import com.intuit.karate.Runner;
+import net.masterthought.cucumber.Configuration;
+import net.masterthought.cucumber.ReportBuilder;
+import org.apache.commons.io.FileUtils;
 import org.junit.BeforeClass;
-import org.junit.runner.RunWith;
+import org.junit.Test;
+import testing.ServerStart;
 
-import com.intuit.karate.junit4.Karate;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
 
-import testing.ServerStart;
+import static org.junit.Assert.assertTrue;
 
 /**
- * TestBase
  *
- * @author thinkerou
+ * @author ericdriggs
  */
-@RunWith(Karate.class)
-public class TestBase {
+// important: do not use @RunWith(Karate.class) !
+public abstract class TestBase {
 
+    protected static final int THREAD_COUNT = 1;
+    protected abstract String getFeatures();
```

Comment:
abstract method should move the file back

Reply:
Move what file?